barretenberg/cpp/src/barretenberg/translator_vm/relation_correctness.test.cpp

```diff
@@ -83,16 +81,12 @@ TEST_F(TranslatorRelationCorrectnessTests, DeltaRangeConstraint)
     prover_polynomials.lagrange_real_last.at(key.dyadic_circuit_size - 1) = 1;
 
     // Create a vector and fill with necessary steps for the DeltaRangeConstraint relation
-    auto sorted_elements_count = (max_value / sort_step) + 1;
-    std::vector<uint64_t> vector_for_sorting(prover_polynomials.ordered_range_constraints_0.size());
-    for (size_t i = 0; i < sorted_elements_count - 1; i++) {
-        vector_for_sorting[i] = i * sort_step;
-    }
-    vector_for_sorting[sorted_elements_count - 1] = max_value;
+    auto sorted_steps = TranslatorProvingKey::get_sorted_steps();
+    std::vector<uint64_t> vector_for_sorting(sorted_steps.begin(), sorted_steps.end());
 
     // Add random values to fill the leftover space
-    for (size_t i = sorted_elements_count; i < vector_for_sorting.size(); i++) {
-        vector_for_sorting[i] = engine.get_random_uint16() & ((1 << Flavor::MICRO_LIMB_BITS) - 1);
+    for (size_t i = sorted_steps.size(); i < vector_for_sorting.size(); i++) {
```

Comment:
Looks like you've maintained the logic from before but pretty sure this loop will never execute since `sorted_steps.size() == vector_for_sorting.size()`. Not sure if something else was intended or what